### DISCUSSION TITLE
PHP Level out of date

### DISCUSSION THREAD

**Discussion by david-unwin** [akeeba/panopticon#624] [Q&A (Community Support)]
When we start Panopticon, we get message '_You are currently using PHP 8.2.17 to run Panopticon. The PHP project has released a newer version, PHP 8.2.18. Please upgrade.'_
However, our hosting povider (IONOS) 
only offers upgrade from 8.2 to 8.3 (see screen shot).
![PHP Settings](https://github.com/akeeba/panopticon/assets/95095238/c75194e6-b401-4bbc-a878-c017852b6ba2)
Should we ignore the message or can we move Panopticon to PHP 8.3?

**Comment by nikosdion** [comment]
Ask your host when they are going to make PHP 8.2.18 available.
